packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/components/SubjectThumbnail/SubjectThumbnail.js

```diff
@@ -0,0 +1,62 @@
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Anchor, Box } from 'grommet'
+import { Media, SpacedText } from '@zooniverse/react-components'
+import styled from 'styled-components'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+const StyledBox = styled(Box)`
+  position: relative;
+`
+const StyledSpacedText = styled(SpacedText)`
+  bottom: 1em;
+  left: 1em;
+  position: absolute;
+  text-shadow: 0 2px 4px rgba(0,0,0,0.5);
+`
+
+function SubjectThumbnail ({ height, href, width, subject }) {
+  const subjectURLs = subject.locations.map(location => Object.values(location)[0])
+  const subjectURL = subjectURLs[0]
+  return (
+    <Anchor
+      href={`${href}/subjects/${subject.id}`}
+    >
+      <StyledBox
+        background='brand'
```

Comment:
Let's remove the background color here... the grid is sizing this div to be slightly larger than the child image and we get the background color showing:

<img width="979" alt="Screen Shot 2019-09-03 at 10 53 02 AM" src="https://user-images.githubusercontent.com/5016731/64188962-08f14c80-ce39-11e9-8ecf-d05099ee45b2.png">

Reply:
Weird, the image should be filling the full width, like the screenshot from the NfN project above.

Reply:
Depends on what the column fractions of the grid calculate to by the browser and the parent div with the background color is just width 100%. I think my laptop window might be at a higher res?

Reply:
I can reproduce this at high resolution when the viewport width is ~1400px or more.
![image](https://user-images.githubusercontent.com/59547/64235543-7ca26080-cef0-11e9-8763-027b2197da38.png)

It's a bug in the Media component, which styles the image with a max height and width so won't fill its parent if the parent dimensions are larger than the height or width props that are passed in. I'll adjust the height and width props, otherwise that the spacing between the images will be wrong at high resolution (with the ugly teal background removed.)